### ISSUE TITLE
Install script may need updates for focal++

### ISSUE THREAD

**Issue by charmarkk** [1337-server/automatic-ripping-machine#177]
- [X] I have checked the wiki
- [X] I have searched for similar issues

**Describe the bug**
The script references PPAs etc. that don't work in focal+ (jammy, impish), though if you ignore those, you can proceed with manual installs. Those PPAs are stebbins/ and mc3mann/ 

It seems like everything is there whether or not you use those particular PPAs (handbrake and abcde still get installed) when following through with manual installation procedures. Are there more reliable sources for those particular pieces of software? It looks like both of those PPAs last got updated with Focal and haven't seen many updates (or logins etc.) since.

Could be a non-issue if the maintainers update, just not sure on that based on their history.

I know jellyfish is _very_ new but impish isn't. Could be they were waiting for the next major release (though impish was, though interim).

**To Reproduce**
1. Attempt to install with script on jammy jellyfish
2. Observe some PPAs error out
3. Try to install manually, after having ran the script once (despite apt errors), skipping the first line referencing mc3mann's PPA.
4. Everything installs with no issues.

### Environment
Ubuntu Jammy Jellyfish
ARM Latest stable

**Comment by shitwolfymakes** [comment]
Sorry it took so long for me to get to this, I do want to provide context for closing this.

ARM is fundamentally built on 20.04. Over the past few years, after the upstream went silent and people made piecemeal contributions, the userbase spread to multiple different versions of Ubuntu, Debian, Gentoo, etc. The docker version was built on Ubuntu 20.04 as well.

At the moment, I am solely focused on recombining the stale docker branch and main thick-client branch into one branch that works for both. I don't have time or resources to test the thick client on multiple versions of multiple different OSs, especially when the docker container is pegged to a single version of a single OS.

Once this work is completed and we don't have to worry about maintaining two versions (or more accurately, letting one die and focusing on the other) of the same codebase, then I'm happy to reconsider supporting multiple OSs and all the OS-specific glitches that come with it. For the moment, however, I don't think it's a tough ask for users to use one OS and version.

Tl;dr: The codebase for ARM is built on pillars of sand that we're finally making progress on shoring up, I can't focus on that when I have to test every fix on a dozen different OS/Version combos.

Hope that helps!

**Comment by charmarkk** [comment]
@shitwolfymakes That makes total sense and I suspected something similar. I've since downgraded back to Focal (Jellyfish still has some other growing pains it seems) so everything is fine for now anyhow. 😊

**Comment by shitwolfymakes** [comment]
Lovely! I don't wanna come off as some `arch btw`-esque distro zealot. I just don't have time for it!
